src/ui/components/results/verticalresultscomponent.js

```diff
@@ -241,7 +242,24 @@ export default class VerticalResultsComponent extends Component {
   getVerticalURL (data = {}) {
     const verticalConfig = this._verticalsConfig.find(config => config.verticalKey === this.verticalKey) || {};
     const verticalURL = this._config.verticalURL || verticalConfig.url || data.verticalURL || this.verticalKey + '.html';
-    return addParamsToUrl(verticalURL, { query: this.query });
+
+    let tabOrder = getDefaultTabOrder(this._verticalsConfig, getUrlParams());
+    let dataTabOrder = [];
+    if (this.core.globalStorage.getState(StorageKeys.NAVIGATION)) {
+      dataTabOrder = this.core.globalStorage.getState(StorageKeys.NAVIGATION).tabOrder;
+    }
+    // We want to persist the params from the existing URL to the new
+    // URLS we create.
+    if (tabOrder !== undefined && dataTabOrder !== undefined) {
+      tabOrder = mergeTabOrder(dataTabOrder, tabOrder, this._verticalsConfig);
```

Comment:
We're passing in this._verticalsConfig as tabs. Are these things interchangeable?

Reply:
I can answer this one, yes they are the same thing! I think one of Alexis' earliest contributions from a whiel back was updating tabConfig to verticalPages (and I think adding some additional properties? Don't remember exactly)